src/olympia/devhub/templates/devhub/addons/submit/describe.html

```diff
@@ -235,13 +235,13 @@ <h3>{{ _('Describe Add-on') }}</h3>
       </div>
     {% endif %}
     <div class="submission-buttons addon-submission-field">
-      <button class="delete-button" type="submit" formnovalidate
-              formaction="{{ url('devhub.addons.cancel', addon.slug) }}">
-          {{ _('Cancel and Disable Version') }}
+      <button type="submit">
+        {{ _('Continue') }}
```

Comment:
does this do the same as the old code? ```{{ _('Submit Version') }}```?

Reply:
Good idea, I copied the wrong copy. Thanks!